sysutils/dvtm/Portfile

```diff
@@ -18,23 +19,31 @@ long_description        dvtm brings the concept of tiling window management, \
                         it easy to work with multiple console based programs \
                         like vim, mutt, cmus or irssi.
 
-homepage                http://brain-dump.org/projects/dvtm/
-master_sites            ${homepage}
+homepage                https://brain-dump.org/projects/dvtm
+master_sites            ${homepage}/
 
-checksums               rmd160  36f14f5e7da72aeee7ca412d905d7d50047208d4 \
-                        sha256  7750872578e259d86c79277c19f3715c1fd3ff8d4f6c300ff8e12b270db86651
+checksums               rmd160  f4222388098891433711020b2739e5756b7c2122 \
+                        sha256  8f2015c05e2ad82f12ae4cf12b363d34f527a4bbc8c369667f239e4542e1e510 \
+                        size    40511
 
 depends_lib             port:ncurses
 
-patchfiles              patch-src-config.mk.diff
+makefile.has_destdir    yes
 
-platform darwin {
-    patchfiles-append   patch-src-Makefile.diff
-}
+build.pre_args-append   PREFIX=${prefix} \
+                        VERSION=${version}
+
+destroot.args           PREFIX=${prefix}
```

Comment:
One of the purposes of the makefile 1.0 portgroup is to set PREFIX for you, so you should not need to set it manually. See https://github.com/macports/macports-ports/blob/master/_resources/port1.0/group/makefile-1.0.tcl#L113-L125. So you should only need to add the project-specific VERSION, and usually you would do so using `build.args`. `*.pre_args` and `*.post_args` are more likely to have defaults set by MacPorts base and/or portgroups and so it's usually safer not to mess with them in a portfile.

Reply:
So I've run into issues where it seemed as if seemingly the PREFIX for some reason was not being set, and I distinctly remember seeing that when making changes here, but I guess I was wrong.  Changes made.